tests/utils/utils.go

```diff
@@ -234,7 +234,77 @@ func SendERC20Destination(
 	Expect(event.Sender).Should(Equal(crypto.PubkeyToAddress(senderKey.PublicKey)))
 	Expect(event.Amount).Should(Equal(bridgedAmount))
 
-	return receipt, event.TeleporterMessageID, event.Amount
+	return receipt, event.Amount
+}
+
+func SendERC20MultihopAndVerify(
+	ctx context.Context,
+	network interfaces.Network,
+	fundedKey *ecdsa.PrivateKey,
+	recipientKey *ecdsa.PrivateKey,
+	recipientAddress common.Address,
+	fromSubnet interfaces.SubnetTestInfo,
+	fromBridge *erc20destination.ERC20Destination,
+	fromBridgeAddress common.Address,
+	toSubnet interfaces.SubnetTestInfo,
+	toBridge *erc20destination.ERC20Destination,
+	toBridgeAddress common.Address,
+	cChainInfo interfaces.SubnetTestInfo,
+	bridgedAmount *big.Int,
+) {
+	teleporterUtils.SendNativeTransfer(
+		ctx,
+		fromSubnet,
+		fundedKey,
+		recipientAddress,
+		big.NewInt(1e18),
+	)
+	input := erc20destination.SendTokensInput{
+		DestinationBlockchainID:  toSubnet.BlockchainID,
+		DestinationBridgeAddress: toBridgeAddress,
+		Recipient:                recipientAddress,
+		PrimaryFee:               big.NewInt(0),
+		SecondaryFee:             big.NewInt(0),
+		AllowedRelayerAddresses:  []common.Address{},
+	}
+
+	receipt, bridgedAmount := SendERC20Destination(
```

Comment:
Nit: could we differentiate the `receipt`s here to make it more clear what each is, and less error to be used incorrectly after it's re-assigned?

Maybe `originReceipt`, `intermediateReceipt`, and `destinationReceipt`?

Reply:
agreed, updated